### PR TITLE
Privates mangling

### PR DESCRIPTION
**Pull request by johnspackman**
This adds mangling for private symbols, which work in source targets as well as build targets.

There is a PR in the framework https://github.com/qooxdoo/qooxdoo/pull/9973 which goes along side this one - now that names are mangled, part of this PR is to make sure that the display name of the function is preserved as the original, and until that PR is merged into the framework you will see some ugly names in the call stack

### CODE REVIEW

**Comment by johnspackman** [comment]
fixes https://github.com/qooxdoo/qooxdoo-compiler/issues/476

File: source/class/qx/tool/compiler/Analyser.js [hunk 1]
```diff
@@ -123,6 +123,12 @@ qx.Class.define("qx.tool.compiler.Analyser", {
       nullable: false,
       check: "Array"
     },
+    
+    /** Whether and how to mangle private identifiers */
+    manglePrivates: {
+      init: "readble",
```
Comment (hkollmann) [inline source/class/qx/tool/compiler/Analyser.js:129]: Here is a typo...

Comment (johnspackman) [reply source/class/qx/tool/compiler/Analyser.js:129]: oh yes - and also a bug that the framework allows it!

**Comment by cboulanger** [comment]
Thank you @johnspackman !

**Pull request merged**